news/2020-07-15-new-featured-artist-numtack05.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "New Featured Artist: Numtack05"
+date: 2020-07-15 08:00:00 +0000
+---
+
+**Numtack05** joins the lineup as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/90/header.jpg)
+
+Making waves in the Vocaloid scene for years with his unique take on instrumentation and composition, **Numtack05** (also known as *Putin-P* and *Teihen-P*) is a Japanese composer and producer who has been gaining attention in the osu! community as of late.
+
+Avid Vocaloid mappers like [Noffy](https://osu.ppy.sh/users/1541323) have been hard at work bringing his creations to life in beatmap form, and we're excited to announce that a selection of his works are now officially nestled under the Featured Artist banner.
+
+**15** tracks from across his sprawling discography are now available, complete with a new showcase map from the ever-talented Mappers' Guild.
+
+Check it all out:
+
+### Genkaku Catastrophe
+
+Check out [this recent set](https://osu.ppy.sh/beatmapsets/675086) by [Noffy](https://osu.ppy.sh/users/1541323)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/90/previews/2374.mp3" type="audio/mpeg">
+</audio>
+
+### Oshimai da ze!
+
+Check out [this Mappers' Guild showcase set](https://osu.ppy.sh/beatmapsets/1001813) by [Arphimigon](https://osu.ppy.sh/users/5392762)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/90/previews/2381.mp3" type="audio/mpeg">
+</audio>
+
+### kimi no hitomi ni
+
+Check out [this collaborative set](https://osu.ppy.sh/beatmapsets/632589) hosted by [Noffy](https://osu.ppy.sh/users/1541323), and [this full-length set](https://osu.ppy.sh/beatmapsets/1103070) from [L3NNy](https://osu.ppy.sh/users/11756041)
```

Comment:
```suggestion
Check out [this collaborative marathon set](https://osu.ppy.sh/beatmapsets/632589) hosted by [Noffy](https://osu.ppy.sh/users/1541323), and [this solo marathon](https://osu.ppy.sh/beatmapsets/1103070) from [L3NNy](https://osu.ppy.sh/users/11756041)
```

L3nny's actually cuts the second song of the video out while arphi and I did both so to call it full length would be backwards

yes arphi made a 7 minute long GD he's a madlad